Pluralize.NET.Tests/TestData/InputData.csv

```diff
@@ -592,6 +591,52 @@ accomplice,accomplices
 amice,amices
 titmouse,titmice
 slice,slices
+base,bases
+database,databases
+rise,rises
+uprise,uprises
+size,sizes
+prize,prizes
+booby,boobies
+hobby,hobbies
+baby,babies
+cookie,cookies
+budgie,budgies
+calorie,calories
+brownie,brownies
+lolly,lollies
+hippie,hippies
+smoothie,smoothies
+techie,techies
+specie,species
```

Comment:
Is this correct? Shouldn't it be an uncountable?

Reply:
Let me double check here to make sure.

Reply:
I found this discussion in Wiktionary https://en.wiktionary.org/wiki/Talk:specie
Also this
https://brians.wsu.edu/2016/05/31/specie/